app/model.py

```diff
@@ -104,10 +104,22 @@ class Repo(db.Model):
     existence of a repository.  Key name: unique repository name.  In the UI,
     each repository behaves like an independent instance of the application."""
 
-    # No properties for now; only the key_name is significant.  The repository
-    # title and other settings are all in ConfigEntry entities (see config.py).
-    # The per-repository 'deactivated' setting blocks UI and API access to the
-    # repository, replacing all its pages with a deactivation message.
+    class Status(object):
```

Comment:
Can you document meaning of each enum value? FYI The repository admin page has detailed documentation of what each state implies.

Then you can delete the comment below:

> The per-repository 'deactivated' setting...

which is now a bit weird to be placed there.

Reply:
Done.